src/components/PublicGroupList.vue

```diff
@@ -193,6 +517,100 @@ export default {
             }
             return "#DC2626";
         },
+
+        /**
+         * Handle browser back/forward button events
+         * @returns {void}
+         */
+        handlePopState() {
+            this.loadSortSettingsFromURL();
+        },
+
+        /**
+         * Load sort settings from URL
+         * @returns {void}
+         */
+        loadSortSettingsFromURL() {
+            if (!this.$root.publicGroupList) {
+                return;
+            }
+
+            const urlParams = new URLSearchParams(window.location.search);
```

Comment:
I think the vue router already gives access to this.

Handling this manually seems like a lot of unnessesary work, especially in the reactivity department above.

I think the code would be much simpler if instead of hooking up events in the right places, we were to use a `computed` to sort and let vue handle the reactivity.